### PR TITLE
Add interactive tool for Mgnify Jupyter lab

### PR DESCRIPTION
**Pull request by bebatut**
## How to test the changes?

- [x] Instructions for manual testing are as follows:
  1. Bring Galaxy's config (galaxy.yml) and job_conf.yml config changes used for running interactive tools (galaxy.yml.interactivetools and job_conf.xml.interactivetools)
   2. Include this tool in the Galaxy's default list of tools (configure in tool conf)

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: tools/interactive/interactivetool_mgnify_notebook.xml [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<tool id="interactive_tool_mgnify_notebook" tool_type="interactive" name="Interactive MGnify Notebook" version="0.0.1" profile="22.01">
+    <requirements>
+        <container type="docker">quay.io/microbiome-informatics/emg-notebooks.dev:latest</container>
```
Comment (bgruening) [inline tools/interactive/interactivetool_mgnify_notebook.xml]: we probably want to pin here a version ...

**Comment by SandyRogers** [comment]
@bebatut @bgruening do you need anything from me to get the CI tests passing?

Also, we now have a `v1.0.1` tag of the notebooks, so could we use that in `tools/interactive/interactivetool_mgnify_notebook.xml:L3`?
Thanks!

**Comment by bgruening** [comment]
@SandyRogers thanks for the hint. Nothing to do here from your side. We will get this into the next Galaxy release.

If you want to update the tool in the future by yourself, you really just need to change the line that you mentioned.

**Review by bgruening — APPROVED** [review]

**Comment by dannon** [comment]
Failed test is unrelated.  Thanks!

**Pull request merged**